README

```diff
@@ -28,7 +28,10 @@ Hypsilophodontids                     33        33       2.9
 Euoplocephalids                       16        16       4.0
 Styracosaurs                          18        18       3.9
 Microceratops                         22        22       4.1
+Compsognaths                          20        20       2.9
```

Comment:
I think this one is the same as line 20 - `Procompsognathids` -- can you check the book? :-)